skore/src/skore/_sklearn/_estimator/feature_importance_accessor.py

```diff
@@ -62,6 +64,14 @@
 }
 
 
+def _function_call_succeeds(func: Callable) -> bool:
+    try:
+        func()
+        return True
+    except Exception:
```

Comment:
Let's specialize to what the `get_feature_names_out` can do.

```suggestion
    except AttributeError:
```